spread.yaml

```diff
@@ -52,7 +52,7 @@ backends:
       system=$(echo "${SPREAD_SYSTEM}" | tr . -)
       instance_name="spread-${SPREAD_BACKEND}-${instance_num}-${system}"
 
-      multipass launch --cpus 2 --disk 20G --memory 2G --name "${instance_name}" "${multipass_image}"
+      multipass launch --cpus 4 --disk 50G --memory 4G --name "${instance_name}" "${multipass_image}"
```

Comment:
Thanks for flagging this. If it's not OK to increase these for all the spread tests, how do I increase them for my spread test only?

Reply:
The increased cpus and memory are probably ok, but 50 GB seems like quite a big ask, is it strictly necessary?

@erinecon as for your question: This setup creates a single multipass VM (with this configuration), and then all spread tests are executed inside it

Reply:
Thanks for the context, @tigarmo !

As for whether 50 GB is strictly necessary, I think @jdkandersson is the one who can really answer that.

Reply:
We had some out of disk issues at 20 GB because a lot of things need to be installed and running Juju takes quite a bit of resources. We could potentially see if we can go from 50 to 40, though the less buffer we provide the higher the risk of issues if something starts taking up more space for some reason

Reply:
I think we might go with 40GB first as that's what the google runners use.

> the less buffer we provide the higher the risk of issues if something starts taking up more space for some reason

Well, we also need to keep these tests from ballooning in size. There's also the possibility that not every test is cleaning up after itself correctly, but if we bump the available disk space too high we'll just be covering the issues.

Reply:
Changed the Multipass VM to 40 GB